docs/static/openapi/edge-api.yaml

```diff
@@ -0,0 +1,225 @@
+openapi: 3.0.3
```

Comment:
Do we need a note saying this is for interaction with our SDK endpoints / link to admin API swagger for other functions?

Reply:
I've added an overview page that makes things clearer here